gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/tasks/CreateManifestTask.java

```diff
@@ -215,8 +215,8 @@ final void createManifest() throws Exception {
             }
 
             try {
-                RecommendedProductDependencies recommendedDeps = jsonMapper.readValue(pdeps.get(),
-                        RecommendedProductDependencies.class);
+                RecommendedProductDependencies recommendedDeps =
```

Comment:
This technically allows users to declare a dependency on themselves... I would move the logic and remove the dependency right before we serialise the manifest

Reply:
Not sure if moving is the right answer, since then users could still declare a dependency on themselves, even if it's then removed.
I'd instead also check the explicitly declared dependencies and throw if any of them is a self dependency.

Reply:
ok, we should that additional verification then